OloEngine/src/OloEngine/Renderer/Debug/CommandPacketDebugger.cpp

```
@@ -1,834 +1,1104 @@
         }
     }
 
-    void CommandPacketDebugger::AnalyzeDrawKeys(const CommandBucket* bucket)
-    {
-        OLO_PROFILE_FUNCTION();
+    // ========================================================================
+    // Sort Analysis Tab
+    // ========================================================================
 
-        if (!bucket)
+    void CommandPacketDebugger::RenderSortAnalysis(const CapturedFrameData* frame)
```

Clang (14.0.6)

[warning] method 'RenderSortAnalysis' can be made static

(readability-convert-member-functions-to-static)